docs/content/configuration/authentication.md

```diff
@@ -52,3 +52,126 @@ docker run -d -v "$($env:ATHENS-STORAGE):/var/lib/athens" `
    -p 3000:3000 `
    gomods/proxy:latest
 ```
+
+## Bazaar(bzr) private repositories
+
+1. Bazaaar config files are located in
+
+  - Unix
+       
+        ~/.bazaar/
+  - Windows
+        
+        C:\Documents and Settings\<username>\Application Data\Bazaar\2.0
+
+  - You can check your location using
+  
+        bzr version
+
+2. There are 3 typical configuration files
+   
+  - bazaar.conf
+    - default config options
+  - locations.conf
+    - branch specific overrides and/or settings
+  - authentication.conf
+    - credential information for remote servers
+
+3. Configuration file syntax
+
+  - \# this is a comment
+  - [header] this denotes a section header
+  - section options reside in a header section and contain an option name an equals sign and a value
+    - EXAMPLE:
+        
+          [DEFAULT]
+          email = John Doe <jdoe@isp.com>
+
+4. Authentication Configuration
+   
+     Allows one to specify credentials for remote servers.
+     This can be used for all the supported transports and any part of bzr that requires authentication(smtp for example).
+     The syntax obeys the same rules as the others except for the option policies which don't apply.
+
+     Example:
+
+       [myprojects]
+       scheme=ftp
+       host=host.com
+       user=joe
+       password=secret
+
+       # Pet projects on hobby.net
+       [hobby]
+       host=r.hobby.net
+       user=jim
+       password=obvious1234
+
+       # Home server
+       [home]
+       scheme=https
+       host=home.net
+       user=joe
+       password=lessobV10us
+
+       [DEFAULT]
+       # Our local user is barbaz, on all remote sites we're known as foobar
+       user=foobar
+
+       NOTE: when using sftp the scheme is ssh and a password isn't supported you should use PPK
+
+       [reference code]
+       scheme=https
+       host=dev.company.com
+       path=/dev
+       user=user1
+       password=pass1
+
+       # development branches on dev server
+       [dev]
+       scheme=ssh # bzr+ssh and sftp are availablehere
+       host=dev.company.com
+       path=/dev/integration
+       user=user2
+
+       #proxy
+       [proxy]
+       scheme=http
+       host=proxy.company.com
+       port=3128
+       user=proxyuser1
+       password=proxypass1
+
+5. Once we've properly setup our authentication we want to share the bazaar configuration directory with the proxy server in order to reuse those credentials.  Below we're setting it as a volume on our proxy container.
```

Comment:
@gcstang is there any way to indent the below code samples and headers (Bash and Powershell)?

Reply:
done

Reply:
done and pushed